manager/src/main/java/org/openremote/manager/event/ClientEventService.java

```diff
@@ -142,6 +152,7 @@ public void configure() throws Exception {
                         .when(header(WebsocketConstants.SESSION_OPEN))
                             .process(exchange -> {
                                 String sessionKey = getSessionKey(exchange);
+                                sessionKeyConnectionTypeMap.put(sessionKey, HEADER_CONNECTION_TYPE_WEBSOCKET);
```

Comment:
Move `SESSION_OPEN`, `SESSION_CLOSE` to `ClientEventService` and send message to `CLIENT_EVENT_QUEUE` with `HEADER_CONNECTION_TYPE` can then handle processing of client connection open/close in a generic way then

Reply:
Sending a `CONNECTED` message back to the client might not make sense for `MQTT`, not sure it is that useful for `Websocket` either really but I would need to check if anything relies on it (don't think the new `core` component does)